service/service_setup.go

```diff
@@ -34,19 +34,16 @@ import (
 	"github.com/pkg/errors"
 )
 
-func (svc *serviceCore) setupLogging() {
-	if svc.log == nil {
-		err := svc.configProvider.Get("logging").PopulateStruct(&svc.logConfig)
-		if err != nil {
-			ulog.Logger(context.Background()).Info("Logging configuration is not provided, setting to default logger", "error", err)
-		}
+var _simpleCtx = context.Background()
```

Comment:
Although, since it is used only for logger, can we have an _svcLogger = ulog.Logger(_simpleCtx)?